buildbot-conf/master.cfg

```diff
@@ -17,37 +17,75 @@
 # Absolute or relative path to buildbot per-project configureations.
 # For local Buildbot installation it is convenient to use the same directory
 # for main master.cfg and projects .conf files
-FP = "./"
+FP = './'
 
 c = BuildmasterConfig = {}
 
 ####### BUILDSLAVES
 
-# import the passwds
-
-# import private
-# reload(private)
+# import the passwords
+import private
+reload(private)
 
 # c['db_url'] = private.mysqlConnection
 
+c['db'] = {
+    'db_url' : "sqlite:///state.sqlite",
+}
+
 # the 'slaves' list defines the set of allowable buildslaves. Each element is
 # a tuple of bot-name and bot-password. These correspond to values given to
 # the buildslave's mktap invocation.
 from buildbot.buildslave import BuildSlave
 
-c['slaves'] = [
-    BuildSlave("cordova-ios-slave", "pass", max_builds=1),
-    BuildSlave("cordova-android-slave", "pass", max_builds=1),
-    BuildSlave("cordova-windows-slave", "pass", max_builds=1),
-    BuildSlave("cordova-win8-slave", "pass", max_builds=1),
-    BuildSlave("cordova-blackberry-slave", "pass", max_builds=1),
-    BuildSlave("cordova-common-slave", "pass", max_builds=3),
-]
+c['slaves'] = []
+
+c['slaves'].extend([
+
+    # used slaves
+    BuildSlave(
+        'cordova-ios-slave',
+        private.slave34Pwd,
```

Comment:
Yes, it seems like we will just need to keep our `master.cfg` defining the same slaves as Apache's `master.cfg`, and not move them to `cordova.conf`. I changed the password constants to be more descriptive, since now they don't need to be moved to `cordova.conf` and don't need to match Apache's `master.cfg`. Only slave names will need to stay the same.